gradebookng/tool/src/webapp/scripts/gradebook-gbgrade-table.js

```diff
@@ -852,12 +852,12 @@ GbGradeTable.renderTable = function (elementId, tableData) {
     columnDefaults: {
       headerSort: false
     },
-    renderHorizontal: "virtual", //enable horizontal virtual DOM
-    selectableRangeAutoFocus:false, //disable auto focus on cell during range selection
+    // renderHorizontal: "virtual", // Disable until https://github.com/olifolkerd/tabulator/pull/4601 is resolved
```

Comment:
Maybe it would be a good idea to add the jira number in the comment?

Reply:
Okay. I will add it. Thanks